openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -138,58 +138,53 @@
 - control_key: AU-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - key: a
-      text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+      text: |      
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
+ 
     - key: b
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-162
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-163
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (3)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-164
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
```

Comment:
To meet AU-5(3), we'll need to document how to rate limit the cluster logging operators. Believe that is not currently possible, hence the OpenShift product Jira.

Control link:
https://nvd.nist.gov/800-53/Rev4/control/AU-5#enhancement-3

Text:

AU-5(3) | RESPONSE TO AUDIT PROCESSING FAILURES \| CONFIGURABLE TRAFFIC VOLUME THRESHOLDSThe information system enforces configurable network communications traffic volume thresholds reflecting limits on auditing capacity and [Selection: rejects; delays] network traffic above those thresholds.
-- | --